.github/workflows/docker-publish.yml

```
@@ -0,0 +1,84 @@
+      - name: Run Docker Scout vulnerability scan
+        uses: docker/scout-action@v1
+        if: github.event_name != 'pull_request'
+        with:
+          command: cves
+          image: ${{ env.IMAGE_NAME }}:latest
+          sarif-file: docker-scout-results.sarif
+          summary: true
+        continue-on-error: true
+
+      - name: Image digest
+        run: echo "Image pushed with digest ${{ steps.meta.outputs.digest }}"
```

actionlint (1.7.10)

[error] property "digest" is not defined in object type {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tag-names: string; tags: string; version: string}

(expression)